### DISCUSSION TITLE
Listing subscriptions alphabetically?

### DISCUSSION THREAD

**Discussion by avekifes** [UltimaHoarder/UltimaScraper#722] [Ideas]
Would it be possible to add an option that makes the scraper list subscriptions in alphabetical order?

**Comment by jp1151419** [comment]
I still haven't figured what determines the scrape order at the moment. Any ideas?

@avekifes As far as searching for the # of a subscription in the window, you can also just copy/paste in notepad and ctrl+f the one you're looking for.
